news/2017-12-21-owc-2017-concludes.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "osu! World Cup 2017 Concludes"
+date: 2017-12-21 10:00:00 +0000
+---
+
+Last weekend, the final circles were clicked, and the victors decided. 32 teams from across the globe whittled down to only 6, and from them, only 1 emerged victorious. After a hard-fought battle, **Poland** emerged victorious as the osu! Champions of 2017 - read on to find out more!
+
+![](/wiki/shared/news/banners/owc_2017_banner.jpg)
+
+This year, 32 teams from across the globe entered and competed in the biggest circle-clicking spectacle that osu! has to offer. From new teams that just entered the fray like Romania to older returning ones like Japan, the tournament saw an overall increase in competitiveness from past years - joined with fiendishly difficult map-pools to test skills of some of osu!'s best players.
+
+The Finals Weekend saw 6 remaining teams – the United States, Poland, the United Kingdom, South Korea, Taiwan, and Germany – all competing for the ultimate title of **osu! Champion**.
+
+In a decisive display of hard-won skill, **Poland** reigned supreme in the final match, taking home the osu! World Cup and the title until next year. If you supported them with a spiffy profile banner, you get to keep yours and broadcast your choice proudly till the next World Cup starts! 
+
+Here's what went down throughout the final battles:
+
+## Finals Weekend
+
+### South Korea vs Taiwan
+
+**South Korea: 7 - Taiwan: 6**
+
+Beginning with South Korea vs Taiwan, the endless rivalry between the two teams grew to a fever pitch, culminating in a colossal tiebreaker after some herculean effort from players on all sides. Pring and Pray from South Korea took turns carrying the team on their shoulders, while Flask and Rucker responded from Taiwan's side in kind.
+
+Poised to lose the tiebreaker-Tengaku in the wake of South Korea's earlier win against Brazil, Taiwan pulled out all the stops, sending Rucker in with a Hidden mod pick on the ultimate match of the set - scraping a hugely respectable 300k score, but still sadly losing the match overall to South Korea 7-6.
```

Comment:
Repeating the final score is redundant and doesn't quite fit here
might wanna mention Yaong's ridiculous performance on Tengaku too

Reply:
redundancy acceptable here due to the fact that the entire paragraph is one encoded block of information, removing the scores makes it harder to ascertain lead without backtracking to beginning of paragraph (which most people wont)